fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,46 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  echo "Migrate c:\opt\td-agent\etc\td-agent.conf"
+  if exist "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" (
+     move /Y "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" "%~dp0..\etc\fluent\td-agent.conf"
+  )
+  echo "Create c:\opt\fluent\var\log\fluent directory"
+  if not exist "%~dp0..\var\log\fluent" (
+     mkdir "%~dp0..\var\log\fluent"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%f "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* directory (e.g buffer)"
+  for /d %%d in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%d "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\etc\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\etc\td-agent\*) do (
+     move /Y %%f "%~dp0..\etc\fluent\"
+  )
+  echo "migrate c:\opt\td-agent\etc\plugin directory"
+  for /d %%d in (%~dp0..\..\td-agent\etc\plugin\*) do (
+     move /Y %%d "%~dp0..\etc\fluent\plugin\"
+  )
+  sc query state=inactive | findstr fluentdwinsvc
+  if !ERRORLEVEL! == 1 (
+    echo "Active fluentdwinsvc service is detected"
+    sc stop fluentdwinsvc
+    timeout /t 5
+  )
+  echo "Ensure remaining files under td-agent"
+  tree %~dp0..\..\td-agent
+
+  @rem ensure to remove remained c:\opt\td-agent directory
+  rmdir /S /Q %~dp0..\..\td-agent
+
+  @rem create symbolic link to c:\opt\fluent (we can't use hardlink for directory)
+  echo "Create symlink c:\opt\td-agent to c:\opt\fluent"
+  mklink /D %~dp0..\..\td-agent %~dp0..\..\fluent
+  echo "Create symlink c:\opt\fluent\var\log\td-agent to c:\opt\fluent\var\log\fluent"
+  mklink /D %~dp0..\..\fluent\var\log\td-agent %~dp0..\..\fluent\var\log\fluent

```

Comment:
We also need the link `/opt/fluent/etc/td-agent` -> `/opt/fluent/etc/fluent` so that we can access the old conf file by `/opt/td-agent/etc/td-agent/td-agent.conf`.

Reply:
Thanks, right thing to do so.